Hackerrank_problems/2D Arrays/solution.cpp

```diff
@@ -0,0 +1,25 @@
+#include <bits/stdc++.h>
```

Comment:
Please add comments to your solution explaining the logic used.
This is *clearly* mentioned in the contributing guidelines.

Reply:
done